rust-src/concordium_base/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- Added PLT `TokenModuleInitializationParameters` CBOR type
```

Comment:
Is there a reason we need `TokenModuleInitializationParameters` for anything at the moment? While it might be very much needed at some point in the rust SDK if we want to support creating plts, the PR description seems to hint that this PR/task is not about creating plts so I wonder if we really need it now (only add code if we need it for the given task/issue).


It would be nice to include in the PR description that this PR is linked to: https://github.com/Concordium/concordium-rust-sdk/pull/294